UI/js/factories/factories.js

```diff
@@ -57,6 +57,33 @@ dashboardApp.factory('Courses', function ($http) {
               result.data.Result.message = 'You seem to have no courses this term.';
             }
             $.each(result.data.Result, function (i, l) {
+              $.each(l.Meeting, function (i, m) {
+                switch (m.Days.substring(0,2)) {
+                case 'Mo':
+                  m.DayCode = 1;
+                  break;
+                case 'Tu':
+                  m.DayCode = 2;
+                  break;
+                case 'We':
+                  m.DayCode = 3;
+                  break;
+                case 'Th':
+                  m.DayCode = 4;
+                  break;
+                case 'Fr':
+                  m.DayCode = 5;
+                  break;
+                case 'Sa':
+                  m.DayCode = 6;
+                  break;
+                case 'Su':
+                  m.DayCode = 7;
+                  break;
+                default:
+                  m.DayCode = 7;
+                }
+              });
```

Comment:
I can't help but think there must be a better way to do this.  Aren't there some JS time functions or libraries that do this already?  If not, rather than using a case statement here, how about putting the days of the week (i.e. "Mo", "Tu", etc.) in an array, then get their position in the array as the day code?

Although some people hate switch statements, I have nothing against them.  However, when each case contains only a single statement, it makes for a lot of extra lines of code.  This just doesn't seem like the right place for a switch.